### ISSUE TITLE
Horde Install Still Failing

### ISSUE THREAD

**Issue by rogathom** [SynoCommunity/spksrc#2333]
Message I receive when running horde installer from http://packages.synocommunity.com/ 

Failed to install "Horde".
Incorrect MySQL root password

Has this package been updated to incorporate the "DSM6: Set MYSQL and MYSQLDUMP variables depending on buildnumber"?

My DSM Version: DSM 6.0-7321 Update 6

( Standalone RoundCube did install )

**Comment by Dr-Bean** [comment]
No, Horde has not been updated yet. Feel free to open a PR to fix the remaining issues with Horde on DSM6.

**Comment by Dr-Bean** [comment]
To be clear, the fixes wrt mysql have been added via [this commit](https://github.com/SynoCommunity/spksrc/commit/d6900764830d2500a382cbd9a27fb84cdaf1dc3a), but that's not all that needs changing for Horde. I believe the PHP command in src/horde.sh needs to be changed as well, and there may be other changes needed as well. So the package on the repository has not been updated yet.

**Comment by kababbles** [comment]
Is there a way to get an updated spk file with the mysql error (noted on mya 20 from rogathom).  Is there a documented method to instll this to synology without the spk?  

thanks in advance,

**Comment by janbro1** [comment]
I see those topics are closed. I tried to update the package (nothing happened), I tried to start the package, nothing happened. What do I do wrong?

**Comment by Dr-Bean** [comment]
Did you miss my comment here: https://github.com/SynoCommunity/spksrc/issues/2333#issuecomment-220635089?

**Comment by janbro1** [comment]
no, did not miss it.

**Comment by Dr-Bean** [comment]
Then maybe you can explain in more detail what you're asking? The current status is that it does not work on DSM6, and there's no update for the Horde package yet to make it compatible.

**Comment by janbro1** [comment]
than why did you close it?

**Comment by Dr-Bean** [comment]
Having duplicate issues open is of no use to anyone.

**Comment by janbro1** [comment]
'xcuse me?!? This topic is open for quite some time, the number of stated bugs alone gives a hint on the severness of the problem. But sure, closing the tickets doesn't help anyone, making the things smaller by decreasing the number, surely does create a level of urgency.
Ok, so where is the bug that deals with this topic?

**Comment by Dr-Bean** [comment]
I'm sorry, it seems there's a bit of misunderstanding here. 

You originally responded in #2487, which is now closed. I closed #2487, because it was a duplicate of _this_ issue, #2333. Both issues dealt with the same problem: Horde does not work with DSM6.
This issue, #2333, is still open, and is used to track the bug with Horde and DSM6. The reason to keep this one open instead of the other one is because this one was opened before the other one, and I have explained the status here.

Having two issues open for the exact same problem is of no use. It just splits off discussion in different locations, instead of concentrating it in one place.

**Comment by janbro1** [comment]
ok, understood, thx for explaining. No offense for the following question, but after all this time, wouldn't it make sense to state the package as abondened?

**Comment by Dr-Bean** [comment]
No offense taken. The question has come up in the past for other packages, and the course of action depends on the circumstances. So here's a question for you: What does declaring a package 'abandoned' mean for you? ;).

The package still works for DSM5, so removing it from the package repository probably isn't the best course of action: enough people are still on that DSM version.
From a technical point of view: Setting packages to be unavailable for DSM6 users is, I believe, possible with a hack, but we probably won't be implementing such a thing. I don't think Synology provides an officially supported method.

On the other hand, you could also say we're simply waiting for someone who is willing to put in the resources to update the package for DSM6 and open a PR. My previous comments in this particular issue might be enough to start off with.

**Comment by hgy59** [comment]
The horde package has been archived and is not supported anymore.
